app/controllers/authors_controller.rb

```diff
@@ -0,0 +1,42 @@
+class AuthorsController < ApplicationController
+
+  def show
+    @author = Author.find(params[:id])
+  end
+
+  def index
```

Comment:
I don't like the actions order. Let's use the following: 

1. index
2. show
3. new
4. edit
5. create
6. update
7. destroy